Jenkinsfile

```
@@ -318,15 +318,16 @@ pipeline {
                   agent { label 'anka' }
                   steps {
                     unstash 'src'
+                    dir('kframework') {
+                      checkout scm
+                    }
```

```suggestion
                    dir('kframework') { checkout scm }
```

Sticking to the formatting where we use less lines has made the Jenkinsfile more readable on many projects.